.ci/wercker.sh

```diff
@@ -92,8 +98,9 @@ no-error-xwiki)
   mvn -f xwiki-commons-tools/xwiki-commons-tool-verification-resources/pom.xml \
     install -DskipTests -Dcheckstyle.version=${CS_POM_VERSION}
   mvn -e test-compile checkstyle:check@default -Dcheckstyle.version=${CS_POM_VERSION}
-  cd ../../
-  rm -rf xwiki-commons
+  find -delete
```

Comment:
please make simply `rm -r .......` , it will fail if folder does not exists.

Reply:
I used `find -delete` and `rmdir`, since it will report failure but never prompt, per @rnveach .  `rm-r` will prompt to remove protected files, of which there are a few in most of these cases.

Reply:
ok, thanks for reminder on why I did `-f` I already forgot.

Unfortunately your approach is not very clear in first look.
Lets keep your approach but wrap it by a shell function `removeFolderWithProtectedFiles hipparchus` (to remove hipparchus folder as example) to make it clear for future reader why removal is not so easy.

this update going out of "minor" ... too much obscure details .
We need to make issue on this to explain our intention, please do. 
Commits should reference issue number. We will later on reuse this approach in other CIs.